src/Catalyst.Core.Lib/IO/Transport/SocketBase.cs

```diff
@@ -62,13 +62,15 @@ protected virtual void Dispose(bool disposing)
             }
             
             _logger.Debug($"Disposing{GetType().Name}");
-
-            var quietPeriod = TimeSpan.FromMilliseconds(100);
-
+            
             try
             {
                 Channel?.Flush();
-                Channel?.CloseAsync().Wait(quietPeriod);
+                
+                // var quietPeriod = TimeSpan.FromMilliseconds(100);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4377003)